src/scripts/notify.sh

```diff
@@ -42,14 +40,18 @@ PostToSlack() {
         echo "Sending to Slack Channel: $i"
         SLACK_MSG_BODY=$(echo "$SLACK_MSG_BODY" | jq --arg channel "$i" '.channel = $channel')
         if [ -n "${SLACK_PARAM_DEBUG:-}" ]; then
-            echo "The message body being sent to Slack is: $SLACK_MSG_BODY"
+            printf "%s" "$SLACK_MSG_BODY" > "$SLACK_MSG_BODY_LOG"
```

Comment:
I'd encourage using a `\n` here or using [the approach described here in "Getting non-clobbered output from command substitution" (which is more general, if more confusing)](https://www.etalabs.net/sh_tricks.html) to guarantee that the file contains a valid POSIX text file, which always has a trailing newline.

`\n` is more straightforward, though, and probably not harmful in this situation.
```suggestion
            printf "%s\n" "$SLACK_MSG_BODY" > "$SLACK_MSG_BODY_LOG"
```